src/common.ts

```diff
@@ -41,6 +53,9 @@ export function baseCommand(commandName: string, formatters: Formatters) {
   const defaultRemote = workspace.getConfiguration('openInGitHub', fileUri).get<string>('defaultRemote') || 'origin';
   const maxBuffer = workspace.getConfiguration('openInGithub', fileUri).get<number>('maxBuffer') || undefined;
   const excludeCurrentRevision = workspace.getConfiguration('openInGitHub').get<boolean>('excludeCurrentRevision') || false;
+  const unsetGitDir = () => workspace.getConfiguration('openInGithub', fileUri).get<boolean>('unsetGitDir');
```

Comment:
For some reason this setting is undefined if we get the value here, and not where we're executing exec.